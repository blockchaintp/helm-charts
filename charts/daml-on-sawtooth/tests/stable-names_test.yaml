suite: daml-on-sawtooth.test for stable names
templates:
  - daml-rpc-service.yaml
  - daml-rpc.yaml
  - serviceaccount.yaml
  - validator-set.yaml
  - validators-svcs.yaml
release:
  name: test
  namespace: testns
tests:
  - it: check for stable entity names (services, statefulsets, serviceaccounts)
    asserts:
      - matchSnapshot:
          path: metadata.name
  - it: check for stable PVC names
    template: daml-rpc.yaml
    set:
      postgres:
        persistence:
          enabled: true
    asserts:
      - matchSnapshot:
          path: spec.volumeClaimTemplates[0].metadata.name
