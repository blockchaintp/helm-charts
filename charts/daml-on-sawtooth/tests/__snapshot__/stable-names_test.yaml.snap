check for stable PVC names:
  1: |
    pgdata
check for stable entity names (services, statefulsets, serviceaccounts):
  1: |
    test-daml-on-sawtooth
  2: |
    test-daml-on-sawtooth-jsonapi
  3: |
    test-daml-on-sawtooth-triggers
  4: |
    test-daml-on-sawtooth-tp
  5: |
    test-daml-on-sawtooth
  6: |
    test-daml-on-sawtooth
  7: |
    test-daml-on-sawtooth-cert
